test/controllers/tickets_controller_test.rb

```diff
@@ -70,6 +71,18 @@ class TicketsControllerTest < ActionController::TestCase
 
   end
 
+  test 'should accept tickets in Base64 encoding' do
+    I18n.locale = :nl
+
+    assert_difference 'ActionMailer::Base.deliveries.size', User.agents.count do
+      assert_difference 'Ticket.count' do
+        post :create, message: @simple_base64_email, format: :json
```

Comment:
This test should fail, because you're not passing the base64: true param, right?